### PR TITLE
Github Page Link

### PR DESCRIPTION
**Pull request by Mikelobi**
### Github Page Link

In this project I:

- **Added Github page to my portfolio**
- **Check Github workflow.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +8,26 @@ This project is to setup a portfolio of mobile version skeleton main page
 ## Built With
 * HTML and CSS
 
+## Live Demo
+
+[Live Demo Link] (https://mikelobi.github.io/Personal-Portifolio/)
+
+
```
Comment (ShoiraTa) [inline README.md:15]: - [ ] This link seems to be broken, please make sure that the application is properly deployed. 
Screenshot 2022-02-28 at 15 44 44

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Mikelobi ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
I can see that you updated the link, but when I follow `https://mikelobi.github.io/Personal-Portifolio/` it still does not work.

**Review by Wes-Isaac — APPROVED** [review]
Hi @Mikelobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**